src/main/java/io/ivyteam/devops/repo/ReposView.java

```diff
@@ -108,7 +108,15 @@ public ReposView(RepoRepository repos, PullRequestRepository prs, BranchReposito
             ScanTypeEnum.DEPENDABOT.getValue()));
       }
       return layout;
-    }).setHeader("Dependabot").setWidth("100px").setSortable(true);
+    }).setHeader("Dependabot").setWidth("100px").setSortable(true)
+        .setComparator((r1, r2) -> {
+          var s1 = securityscanners.getByRepoAndScantype(r1.name(),
```

Comment:
uh this is very expensive. for each and every comparision, you make to more data base request. depending on the list size, this can cause 1000 of database requests.

I quickly will have a look if we can do it another way